api/CanvasRenderingContext2D.json

```diff
@@ -1122,7 +1122,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/currentTransform",
           "support": {
             "chrome": {
-              "version_added": true,
+              "version_added": "32",
```

Comment:
Can you also mark it as removed in 68 based on https://storage.googleapis.com/chromium-find-releases-static/26e.html#26ea97726ef213927ed86a06836dd16a8b33275f?

I expect we can eventually delete this entry entirely, but might as well make this update to make that more clear.

Reply:
Thanks for catching this!  There's a lot of cases I need to watch out for in the commit history, haha...  😅 

Yeah, it looks like this qualifies as an irrelevant feature!

Reply:
I opened up #8486 to remove the entire feature.